tests/integration/test_version_command.py

```diff
@@ -299,3 +388,16 @@ def test_command_raises_exception_invalid_paths():
 
     assert result.exit_code != 0
     assert "does not contain a dbt project" in result.stdout
+
+
+def test_version_no_model_yaml(project):
+    """Verify that the version command running in default mode adds versions and creates a second version, too"""
```

Comment:
can you add an assertion that we get two versions of the model here? and/or update the comment to say "just test that the command runs in default mode"

Reply:
Fantastic call. Will add.